optimum/habana/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion.py

```diff
@@ -495,18 +495,27 @@ def __call__(
 
             # 8. Denoising loop
             throughput_warmup_steps = kwargs.get("throughput_warmup_steps", 3)
+            logger.info(f"{throughput_warmup_steps} warmup steps will be used in measuring throughput")
```

Comment:
I don't think we should log that, I think it would be confusing for users

Reply:
Sounds good, removed the logger info